src/ios/RealmJS.xcodeproj/project.pbxproj

```diff
@@ -246,7 +246,6 @@
 			children = (
 				F6BCCFDF1C83809A00FE31AE /* lib */,
 				F62A35131C18E6E2004A917D /* iOS */,
-				F60103051CC4ADE500EC01BA /* JS */,
```

Comment:
That abstraction was not clear at all. If we make all of are naming consistent I am ok with whatever but as it was I was having to look around in different groups to fine `js_` files which is confusing and annoying.